chart/README.md

```diff
@@ -541,14 +541,64 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
+cluster to automatically scale up/down based on the resource demand such as CPU and Memory utilization.
+
+Note: When setting auto-scaling parameters please make sure to have sufficient hardware resources available in the
+underlying nodes meet the max pod demands.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`, and set
+the `hpa.resources.memory.enabled` parameter to `true` to enable memory-based autoscaling.
+
+   ```
+   --set hpa.enabled=true
+   --set hpa.resources.memory.enabled=true
+   ```
```

Comment:
Lets add a  **For example** pre-heading to this as the memory argument is optional and we don't want users to assume that is mandatory

Reply:
Removed: commit: a5f2f4e88459cc6b5845096a1615dd6d2ead03a9